app/services/search/radius_suggestions_builder.rb

```diff
@@ -19,7 +21,7 @@ def get_radius_suggestions
         [
           wider_radius,
           Search::AlgoliaSearchRequest.new(
```

Comment:
This can probably fit on the same line a bit more nicely now the helper is included